File: src/main/kotlin/space/davids_digital/kiri/rest/dto/telegram/TelegramChatMetadataUpdateRequest.kt
package space.davids_digital.kiri.rest.dto.telegram

import space.davids_digital.kiri.rest.dto.TelegramChatMetadataDto

data class TelegramChatMetadataUpdateRequest (
    val notificationMode: TelegramChatMetadataDto.NotificationMode?,
    val enabled: Boolean?
)

File: admin-ui/lib/api/types/TelegramChatMetadataUpdateRequest.ts
import {TelegramChatMetadataDto} from "@/lib/api/types/TelegramChatMetadataDto";

export type TelegramChatMetadataUpdateRequest = {
    chatId: number;
    notificationMode?: TelegramChatMetadataDto.NotificationMode;
    enabled?: boolean;
}